dtest_setup.py

```diff
@@ -439,7 +440,7 @@ def init_default_config(self):
         phi_values = {'phi_convict_threshold': 5}
 
         # enable read time tracking of repaired data between replicas by default
-        if self.cluster.version() >= '4':
+        if self.cluster.cassandra_version() >= '4':
```

Comment:
Don't you also need to make that change in L98?

Reply:
L98 is
```
        if self.cluster.version() < LooseVersion('3.11.13'):
```
in `def install_nodetool_legacy_parsing`

i don't think so…  what were your thoughts here ?

Reply:
I am obviously missing what is the difference of comparing against a version number and a `LooseVersion`. Aren't you trying to change behavior depending on the 'embedded' underlying C* versions instead of the 'product' version?